bin/do_deltas.py

```diff
@@ -90,6 +90,16 @@ def cont_fit(data):
     parser.add_argument('--multiplicative-flux-calibration',type = str,default=None,required=False,
```

Comment:
I should have caught this before, but can you change this option to --flux-calib ? It's too long now...